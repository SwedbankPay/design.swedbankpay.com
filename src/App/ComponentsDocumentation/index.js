import React from "react";

import { RenderPage } from "../utils";

const ComponentsDocumentation = () => (
    <RenderPage path="/components" initPath="/components/components" />
);

export default ComponentsDocumentation;
