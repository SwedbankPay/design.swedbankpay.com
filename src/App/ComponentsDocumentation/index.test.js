import React from "react";
import { shallow } from "enzyme";

import ComponentsDocumentation from "./index";

describe("ComponentsDocumentation: index", () => {
    it("is defined", () => {
        expect(ComponentsDocumentation).toBeDefined();
    });

    it("renders", () => {
        const wrapper = shallow(<ComponentsDocumentation />);

        expect(wrapper).toMatchSnapshot();
    });
});
