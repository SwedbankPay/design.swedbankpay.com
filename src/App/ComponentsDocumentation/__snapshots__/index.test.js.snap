// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ComponentsDocumentation: index renders 1`] = `
<RenderPage
  initPath="/components/components"
  path="/components"
/>
`;
